controllers/clusters/redis_controller.go

```diff
@@ -869,9 +869,22 @@ func (r *RedisReconciler) newWatchStatusJob(redis *v1beta1.Redis) scheduler.Job
 			}
 		}
 
-		if iRedis.Status.CurrentClusterOperationStatus == models.NoOperation &&
+		equals := redis.Spec.IsEqual(iRedis.Spec)
```

Comment:
please add this logic to the rest `Kind`s

Reply:
already fixed